packages/fx-core/src/plugins/solution/fx-solution/debug/provisionLocal.ts

```diff
@@ -253,6 +271,25 @@ export async function setupLocalEnvironment(
           }
         }
       }
+    } else if (inputs.platform === Platform.VS) {
```

Comment:
Looks like the similar logic as above, is that possible to refactor it in the future?